test/index.js

```diff
@@ -1,12 +1,13 @@
 var assert = require('assert');
 var test = require('testit');
 var parser = require('../');
-var parse = parser;
+var parse = parser.parse;
+var StackElement = parser.StackElement;
 
 test('parse', function () {
   test('works out how much depth changes', function () {
     var state = parse('foo(arg1, arg2, {\n  foo: [a, b\n');
-    assert.deepEqual(state.stack, [ ')', '}', ']' ]);
+    assert.deepEqual(state.stack, [ StackElement.ROUND_BRACKET, StackElement.CURLY_BRACKET, StackElement.SQUARE_BRACKET ]);
```

Comment:
I do think the older style would be more easily consumed in vanilla JavaScript. Introspection of the `state` object through the Node REPL would be more difficult with `state.stack` being an array of numbers too.